### PR TITLE
Fix Go To Webinar email validation

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:

If form with GTW action has different email form field alias like `email` it make issue and validation message is not display in form. This PR fixed it

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Setup GTW
3. Create form with GTW subscription action. Email form field name should be `Email profesional` for example
4. Try submit / register to GTW twice.
5. See second time you don't see validation error , event If we expect see validation error Email already registered to webinar

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11757"></a>

### CODE REVIEW

File: plugins/MauticCitrixBundle/EventListener/FormSubscriber.php [hunk 1]
```diff
@@ -600,4 +599,18 @@ public function onFormBuilder(Events\FormBuilderEvent $event)
             }
         }
     }
+
+    /**
+     * @param Collection<int,Field> $fields
+     */
+    protected function getEmailFieldAlias(Collection $fields): string
```
Comment (escopecz) [inline plugins/MauticCitrixBundle/EventListener/FormSubscriber.php]: Any reason for protected method?
```suggestion
    private function getEmailFieldAlias(Collection $fields): string
```

**Review by escopecz — COMMENTED** [review]
The code change makes sense, thanks! Just one question.

**Review by volha-pivavarchyk — COMMENTED** [review]
Works partly well. A visitor registers once. The second registration is not saved in the contact timeline. But the form freezes when I submit the form a second time. I tested for two contacts.
image
image
image
image

The error is as follows. Maybe the problem is in the integration and not related to this PR.
```
[2023-06-16 14:15:43] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function getStatusCode() on array" at /var/www/html/plugins/MauticCitrixBundle/Api/CitrixApi.php line 58 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getStatusCode() on array at /var/www/html/plugins/MauticCitrixBundle/Api/CitrixApi.php:58)"} []

**Comment by kuzmany** [comment]
I think we remove citrix from  the plugins for 5.x. Close

**Pull request closed without merging**